examples/dashboard_embeddable_examples/public/plugin.tsx

```diff
@@ -1,52 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License
- * 2.0 and the Server Side Public License, v 1; you may not use this file except
- * in compliance with, at your election, the Elastic License 2.0 or the Server
- * Side Public License, v 1.
- */
-
-import { AppMountParameters, AppNavLinkStatus, CoreSetup, Plugin } from '@kbn/core/public';
-import { DashboardStart } from '@kbn/dashboard-plugin/public';
-import { DeveloperExamplesSetup } from '@kbn/developer-examples-plugin/public';
-import { EmbeddableExamplesStart } from '@kbn/embeddable-examples-plugin/public/plugin';
-
-interface SetupDeps {
-  developerExamples: DeveloperExamplesSetup;
-}
-
-interface StartDeps {
-  dashboard: DashboardStart;
-  embeddableExamples: EmbeddableExamplesStart;
-}
-
-export class DashboardEmbeddableExamples implements Plugin<void, void, {}, StartDeps> {
```

Comment:
This was a very old example plugin that I **think** was meant to demonstrate portable dashboards, but I only noticed it after I'd already constructed the `portable_dashboards_example` plugin. So, I just decided to remove the old one since the new naming is more in-line with the recent terminology (portable dashboard VS dashboard embeddable), as well :shrug: